.github/workflows/auto-update-deps.yaml

```diff
@@ -39,7 +39,7 @@ on:
         default: "false"
       branch:
         description: "Branch Name? (empty for default branch of the respective repository)"
-
+      
```

Comment:
Don't think we want the ws in this file

Reply:
This is from the template - I don't want to fight the whitespace in this PR

Reply:
weird I don't see any ws in the template...